test/verify/check-users

```diff
@@ -69,7 +69,7 @@ def createUser(
     run_assert_pixels=False,
 ):
     if default_shell is None:
-        default_shell = getUserAddDetails(machine)["SHELL"]
+        default_shell = getUserAddDetails(machine)["SHELL"] or "/bin/bash"
```

Comment:
Was this neccesseray because of some test failure?

Reply:
Yes, it was appearing in the tests empty for some distros.

Reply:
Yes, Debian-based distros don't user `useradd`, but `adduser`, which has its own config. The default shell there is /bin/sh which is dash, not bash. I haven't checked whether this makes sense (presumably the code has that fallback somewhere as well?).